ratings/ratings.py

```diff
@@ -110,10 +108,6 @@ def import_files(files, bucket=None):
 
             m_num_w = re.match(MODEL_REGEX, pw).group(1)
             m_num_b = re.match(MODEL_REGEX, pb).group(1)
-            # v10 and v9 have the same model and name for 588, so cross eval
-            # games played with this model should be ignored.
-            if m_num_w == '000588' or m_num_b == '000588':
```

Comment:
there are at least 3 other broken instances, verifying that #816 is a no-op was stupidly hard.